bot/exts/recruitment/talentpool/_review.py

```diff
@@ -210,6 +219,13 @@ async def archive_vote(self, message: PartialMessage, passed: bool) -> None:
                 colour=colour
             ))
 
+        # Thread channel IDs are the same as the message ID of the parent message.
+        nomination_thread = message.guild.get_thread(message.id)
+        if not nomination_thread:
+            log.warning(f"Could not find a thread linked to {message.channel.id}-{message.id}")
+            return
+        await nomination_thread.edit(archived=True)
+
         for message_ in messages:
             await message_.delete()
```

Comment:
Should we move this above the archive logic to make sure the nomination message(s) are deleted even if the thread is lost somewhere?

Reply:
Yea, agreed. we should probably wrap the edits and delete in suppress too

Reply:
https://github.com/python-discord/bot/pull/1928/commits/df38d2817bf20f6eac1cde8fa7f63d741f4a5d74